packages/block-library/src/button/block.json

```diff
@@ -61,8 +61,11 @@
 		"anchor": true,
 		"align": true,
 		"alignWide": false,
+		"color": {
+			"__experimentalSkipSerialization": true
+		},
 		"reusable": false,
-		"__experimentalSelector": ".wp-block-button > a"
+		"__experimentalSelector": ".wp-block-button__link"
```

Comment:
It seems that we added the block selector in this PR https://github.com/WordPress/gutenberg/pull/25768 although I haven't it found in use anywhere, so I've reduced its specificity to the minimum.

Note that if we wanted to use the existing selector we'd need to fix https://github.com/WordPress/gutenberg/issues/29381 first.

Reply:
I don't have objections here, the simpler the selector the better it seems.